src/ceph.in

```diff
@@ -886,18 +886,15 @@ def main():
         if not timeout:
             timeout = 5
 
-        hdr('Monitor commands:')
-        print('[Contacting monitor, timeout after %d seconds]' % timeout)
-
-    if childargs and childargs[0] == 'ping':
+    if childargs and childargs[0] == 'ping' and not parsed_args.help:
```

Comment:
fix "ceph ping --help" problem. We should list commands only with "ping"  prefix. 

```
 Monitor commands:
 =================
"ping" requires a monitor name as argument: "ping mon.<id>"
```